package.json

```diff
@@ -39,7 +39,7 @@
     "date-fns-tz": "^2.0.1",
     "form-data": "^4.0.0",
     "friendly-challenge": "^0.9.13",
-    "govuk-frontend": "^5.0.0",
+    "govuk-frontend": "^5.5.0",
```

Comment:
Do we know what changed in this version? Will it mean our pages don;t quite match the prototypes anymore?